charts/replicated-library/templates/lib/_container.tpl

```diff
@@ -45,8 +45,8 @@
 {{- end }}
   {{- with $containerValues.env }}
   env:
-    {{- get (fromYaml (include "replicated-library.env_vars" .)) "env" | toYaml | nindent 4 -}}
-  {{- end }}
+       {{- get (fromYaml (include "replicated-library.env_vars" .)) "env" | toYaml | nindent 4 -}}
```

Comment:
just a aesthetic thing here but this indentation is off

Reply:
fix indentation.